src/lib/localStore.ts

```diff
@@ -113,6 +116,22 @@ export default class ExtensionStore {
     );
   }
 
+  getState(keys?: string | string[]) {
```

Comment:
May be use spread for argument? So this can be simplified to e.g.:
```typescript
getState(...keys?: string[]){
  if(keys.length === 0) return this._state;
  
  return keys.reduce(
      (acc, key) =>
        this._state[key] ? { ...acc, [key]: this._state[key] } : acc,
      {}
    );
}
```

Reply:
The argument cannot be optional, and the typescript also complains because a string is not expected as an argument.

Reply:
Ofс, spread arguments are always an array, and it should be `getState(...keys: string[])`.